packages/commonwealth/client/scripts/views/components/sidebar/index.tsx

```diff
@@ -47,47 +45,43 @@ export const Sidebar = () => {
 
   return (
     <div className="Sidebar">
-      {menuName === 'default' && (
-        <div className="sidebar-default-menu">
-          <SidebarQuickSwitcher />
-          {app.chain && (
-            <div className="community-menu">
-              {showAdmin && <AdminSection />}
-              {featureFlags.communityHomepage && app.chain.meta.hasHomepage && (
-                <div
-                  className={onHomeRoute ? 'home-button active' : 'home-button'}
-                  onClick={() => navigate('/feed')}
-                >
-                  <CWIcon iconName="home" iconSize="small" />
-                  <CWText>Home</CWText>
+      <div className="sidebar-default-menu">
+        <SidebarQuickSwitcher />
```

Comment:
the commonwealth sidebar will now not cause the ui glitch when clicked in `community` layout pages